app/projects/skilled-elearning-landing-page/components/NavBar.jsx

```diff
@@ -0,0 +1,59 @@
+'use client';
+
+import React from 'react';
+import { styled } from 'styled-components';
+import { theme } from '../styles';
+import { LogoDark } from '../icons';
+import Button from './Button';
+
+const StyledNavBar = styled.nav`
+  width: 100%;
+  height: 7rem;
+  background-color: transparent;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 100;
+
+  & .container {
+    max-width: 1110px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    @media only screen and (max-width: 1279px) {
+      max-width: 768px;
+    }
+
+    @media only screen and (max-width: 764px) {
+      width: 343px;
+    }
+
```

Comment:
This class is used in many places, could you create a Container global componente for reusability?